ultimatejobweb/views.py

```diff
@@ -34,4 +34,4 @@ def sign_in(request):
 
 
 def available_jobs(request):
-    return render(request, r"available_jobs.html", {'job': job})
+    return render(request, r"available_jobs.html", {'Job': Job.objects.all()})
```

Comment:
# Please consider the following:
jobs = Job.objects.all()
return render(request, r"available_jobs.html, {
            'jobs_list': jobs })